arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -1626,7 +1626,7 @@
 		4C72116F20BF444C004A7DD9 /* SVProgressAnimatedView.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = SVProgressAnimatedView.h; sourceTree = "<group>"; };
 		4C72117120BF4456004A7DD9 /* SVRadialGradientLayer.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = SVRadialGradientLayer.h; sourceTree = "<group>"; };
 		4C72117220BF4457004A7DD9 /* SVRadialGradientLayer.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = SVRadialGradientLayer.m; sourceTree = "<group>"; };
-		4C73C83421112B4300B0F9C7 /* ArcGIS.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = ArcGIS.framework; path = $HOME/Library/SDKs/ArcGIS/iOS/Frameworks/Dynamic/ArcGIS.framework; sourceTree = "<absolute>"; };
+		4C73C83421112B4300B0F9C7 /* ArcGIS.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = ArcGIS.xcframework; path = $HOME/Library/SDKs/ArcGIS/Frameworks/ArcGIS.xcframework; sourceTree = "<absolute>"; };
```

Comment:
![image](https://user-images.githubusercontent.com/9660181/98980071-ffe61b80-24d0-11eb-9f83-24fa7eff5562.png)

I think the path should be absolute, but in toolkit it is set to "Relative to Group".

Reply:
You are correct; it should be absolute. That should be corrected in the toolkit.

Reply:
OK. Let me create a PR to fix that.

Reply:
I don't have write access to toolkit repo. Let me create an issue instead.

Reply:
I can fix it.

Reply:
The issue has been [fixed](https://github.com/Esri/arcgis-runtime-toolkit-ios/pull/126).